lib/ncurses.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<interface xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd" uri="https://apps.0install.net/lib/ncurses.xml" xmlns="http://zero-install.sourceforge.net/2004/injector/interface">
+  <name>ncurses</name>
+  <summary xml:lang="en">terminal-independent  text-based user interface library</summary>
+  <description xml:lang="en">The ncurses (new curses) library is a free software emulation of curses in System V Release 4.0 (SVr4), and more. It uses terminfo format, supports pads and color and multiple highlights and forms characters and function-key mapping, and has all the other SVr4-curses enhancements over BSD curses. SVr4 curses became the basis of X/Open Curses.</description>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon" />
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.png" type="image/png" />
+  <category>Development</category>
+  <homepage>https://sourceforge.net/projects/ezwinports/files/</homepage>
+  <needs-terminal />
+  <implementation arch="Windows-i486" version="5.9" released="2012-04-07" license="MIT/X Consortium License, LGPL (GNU Lesser General Public License)" doc-dir="share/man" id="sha1new=68968061b2fbd255688172377f4a0778515b5855">
+    <command name="run" path="bin/captoinfo.exe" />
```

Comment:
I think it would be better to rename this command from `run` to `captoinfo`.

Since this feed is a library first and only provides associated tools second, I think it's fine if it cannot be used with `0install run` without explicitly specifying a `--command`.

Reply:
renamed